cfy_manager/components/prometheus/prometheus.py

```diff
@@ -201,7 +198,7 @@ def configure(self, upgrade=False):
         self.start()
 
     def upgrade(self):
-        self.configure(upgrade=True)
+        _update_manager_alerts_services()
```

Comment:
```suggestion
        _update_manager_alerts_services()
        self.start()
```

Reply:
why though? the upgrade in main.py runs component.start anyway

Reply:
for the exporters